client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import StepWrapper from 'signup/step-wrapper';
+import Gridicon from 'components/gridicon';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+import renderHeroImage from './hero-image';
+
+export default React.createClass( {
+	displayName: 'PressableStoreStep',
+
+	propTypes: {
+		stepName: React.PropTypes.string.isRequired,
+		goToNextStep: React.PropTypes.func.isRequired,
+		signupDependencies: React.PropTypes.object.isRequired,
+	},
+
+	getInitialState() {
+		return {
+			email: '',
+		};
+	},
+
+	getDefaultProps() {
+		return {};
+	},
+
+	componentDidMount() {
+		if ( this._input ) {
+			this._input.focus();
+		}
+	},
+
+	onEmailChange( event ) {
+		this.setState( {
+			email: event.target.value,
+		} );
+	},
+
+	onSubmit( event ) {
+		window.location.href = `https://my.pressable.com/signup/five-sites?email=${ encodeURIComponent( this.state.email ) }&wp-ecommerce=true`;
+		event.preventDefault();
+	},
+
+	renderStoreForm() {
+		return (
+			<div>
+				<LoggedOutForm className="pressable-store__form">
+					{ renderHeroImage() }
+
+					<FormSectionHeading className="pressable-store__heading">{ this.translate( 'Create your new WordPress store for as low as $20.83/Month' ) }</FormSectionHeading>
+					<p className="pressable-store__copy">{ this.translate( 'WordPress.com has partnered with hosting provider Pressable to make setting up your store on WordPress simple and fun.' ) }</p>
+
+					<LoggedOutFormFooter>
+						<FormLabel for="email">{ this.translate( 'Enter your email address to get started:' ) }</FormLabel>
+						<div className="pressable-store__form-fields">
+							<FormTextInput ref={ ( input ) => this._input = input } onChange={ this.onEmailChange } className="pressable-store__form-email is-spaced" type="email" placeholder="Email Address" name="email" />
```

Comment:
@aduth @mtias The linter has complained about this, however this is one of the recommended approaches as [per the React docs](https://facebook.github.io/react/docs/more-about-refs.html). Should I use an instance method defined on the react class body instead?